uReplicator-Common/src/main/java/com/uber/stream/kafka/mirrormaker/common/core/KafkaBrokerTopicObserver.java

```diff
@@ -75,7 +78,19 @@ public KafkaBrokerTopicObserver(String brokerClusterName, String zkString) {
       public void run() {
         tryToRefreshCache();
       }
-    }, 0, 60, TimeUnit.SECONDS);
+    }, 0, refreshTimeIntervalInMillis, TimeUnit.MILLISECONDS);
+    registerMetric();
+  }
+
+  private void registerMetric() {
```

Comment:
Maybe add a refresh failure counter to monitor failed refresh.

Reply:
done